sys/crypto/des/des.h

```diff
@@ -65,7 +65,7 @@ typedef struct des_ks_struct
 	des_cblock cblock;
 	/* make sure things are correct size on machines with
 	 * 8 byte longs */
-	DES_LONG deslong[2];
+	DES_LONG deslong[2] __subobject_use_container_bounds;
```

Comment:
Can we just annotate the uses of this field and/or change `&foo->deslong[0]` -> `foo->deslong`

Reply:
Yes, the rest of the uses seem safe.